tests/pom.xml

```diff
@@ -252,7 +252,7 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     </log>
                     <wait>
                       <log>^.*(Started \w*Application in).*$</log>
-                      <time>30000</time>
+                      <time>40000</time>
```

Comment:
This has nothing to do with the other changes and should therefore go into a separate commit.